### PR TITLE
create github documentation and sample .github folders

### PR DESCRIPTION
**Pull request by soleilyasmina**
This implementation adds info about github and how to use Community Health files in the .github folder, among others.

### CODE REVIEW

File: docs/github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- `chore/name-of-branch`: refactoring or updating existing code
+- `docs/name-of-branch`: adding documentation
+- `feat/name-of-branch`: creating a feature
+- `fix/name-of-branch`: fixing a bug
+
+## Making Pull Requests
+
+All pull requests apart from deployment should be made from the feature branch to `main`.
+
+Before making a pull request, please make sure to rebase your branch with `main` to ensure all previous features are included.
```
Comment (cpresler) [inline docs/github/CONTRIBUTING.md]: It should be clear that rebase should be from origin/main, and that you need to fetch before you rebase

Comment (soleilyasmina) [reply docs/github/CONTRIBUTING.md]: Gotcha! Adding it now.

File: docs/github.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Github
+
+## Table of Contents
+
+## Automation with Issue Ticket and Pull Request Templates
+
+GitHub provides the ability to generate pull request and issue ticket templates to encourage consensus of code style, purpose of changes and structuring issues. 
+
+This folder provides examples of a pull request template, a bug issue ticket template, and a feature issue ticket template.
```
Comment (cpresler) [inline docs/github.md]: Linking the to the various templates might be helpful

**Comment by cpresler** [comment]
approved.

**Pull request merged**